src/logic/claims/data.ts

```diff
@@ -132,20 +129,16 @@ async function loadClaims(
 
   const [claims, disputes] = await Promise.all([
     getClaimContractData(claimsManager, createdEvents),
-    getDisputeContractData(
-      arbitratorProxy,
-      // TODO DAO-189 Remove filter. See https://github.com/api3dao/claims-manager/issues/43
-      disputeEvents.filter((ev) => ev.args.disputeId.gt(0))
-    ),
+    getDisputeContractData(arbitratorProxy, disputeEvents),
   ]);
 
   // Combine the static and dynamic data
   const claimsById = createdEvents.reduce((acc, event, index) => {
     const eventArgs = event.args;
-    const claimId = eventArgs.claimIndex.toString();
+    const claimId = eventArgs.claimHash;
```

Comment:
Should we call it claimHash in the app as well? Probably not worth the hassle right?

Reply:
Yeah I also don't think it's worth it 👍